clm/lm_wrapper.py

```diff
@@ -13,7 +13,7 @@
 from qanta.util.environment import QB_QUESTION_DB, QB_WIKI_LOCATION
 from qanta.util.constants import CLM_PATH, QB_SOURCE_LOCATION
 
-from clm import clm
+import clm
```

Comment:
I'll look more into this, its a bit strange. If an issue isn't open already can you open one?